src/settings-ui/Settings.UI.Library/PowerLauncherProperties.cs

```diff
@@ -72,6 +72,9 @@ public class PowerLauncherProperties
         [JsonPropertyName("search_wait_for_slow_results")]
         public bool SearchWaitForSlowResults { get; set; }
 
+        [JsonPropertyName("should_use_pinyin")]
+        public bool ShouldUsePinyin { get; set; }
```

Comment:
"should" makes it feel strange, as if you never know what's going to happen. I would suggest `use_pinyin`